docs/pages/build/framework/action/serialized-action.mdx

```diff
@@ -44,3 +45,23 @@ Returns the execution logs for the action. If not null, the execution succeeded
 Returns any errors that occurred during the execution of the action. It is null if the action has not yet been executed. Here the action is of type ``SerializedAction``.
 
 **Type:** `ExecutionError[] | null;`
+
+## Serialized Block
+
+In some places, the `SerializedAction` also includes block details. Then its datatype is `SerializedActionAndBlock` as given below
+
+```ts
+interface SerializedBlock {
+  height: number;
+  hash: string;
+  timestamp: number;
+  status: BlockStatus;
+  batchInfo?: BatchInfo;
+}
+
+interface SerializedActionAndBlock extends SerializedAction {
```

Comment:
awkward naming convention `SerializedActionWithBlock` is better than `SerializedActionAndBlock`
can take this up later in sdk, ok for now